Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
Given the deadline on this, If you want to merge this and discuss the potential of `brew repair-permission` over the next few days/weeks/whatever I'd be happy. I don't know if we'll have time to bash out & be happy with a new implementation before El Cap starts becoming available later today.